ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--
+
+Thanks for opening an issue! To help the libGDF team handle your information
+efficiently, please first ensure that there is no other issue present that
+already describes the issue you have
+(search at https://github.com/gpuopenanalytics/libgdf/issues?&q=is%3Aissue).
+
+If there is no issue present please jump to a section below and delete the
+irrelevant one depending on whether you are:
+
+ * Making a feature request.
+ * Reporting a bug.
+
+For more general "how do I do X?" type questions, please speak visit
+http://gpuopenanalytics.com/#/COMMUNITY for links to Slack and Google
+Groups.
+
+-->
+
+## Feature request
+
+<!--
+
+Please include details of the feature you would like to see, why you would
+like to see it/the use case
+
+-->
+
+## Reporting a bug
+
+<!--
+
+Before submitting a bug report please ensure that you can check off these boxes:
+
+-->
+
+- [ ] I am using the latest version of libGDF from conda or built from `master`.
+- [ ] I have included the following environment details:
+      Linux Distro, Linux Kernel, GPU Model
+- [ ] I have included the following version information for:
+      Arrow, CUDA, Numpy, Pandas, Python
```

Comment:
I'm not sure how relevant Numpy / Pandas are here. I think we can replace Numpy and Pandas with GCC version and CMake version as a starting point.